mods/ra/weapons/missiles.yaml

```diff
@@ -338,8 +362,13 @@ SCUD:
 			Heavy: 40
 			Concrete: 100
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath, Incendiary
-	Warhead@3Eff: CreateEffect
+	Warhead@EffDirectHit: CreateEffect
 		Explosions: napalm
 		ImpactSounds: firebl3.aud
-	Warhead@4EffWater: CreateEffect
+	Warhead@EffGround: CreateEffect
+		Explosions: napalm
+		ImpactSounds: firebl3.aud
+	Warhead@EffSubmergedHit: CreateEffect
+		Explosions: large_splash
+	Warhead@EffWater: CreateEffect
```

Comment:
Why does this have no
```
ValidTargets: 
InvalidTargets:
```
?

Reply:
`^SubMissileDefault` inherits them from `^AntiGroundMissile`.